vignettes/openxlsx2_formulas_manual.Rmd

```diff
@@ -89,8 +89,8 @@ m1 <- matrix(1:6, ncol = 2)
 m2 <- matrix(7:12, nrow = 2)
 
 wb <- wb_workbook()$add_worksheet()$
-  add_data(x = m1, startCol = 1)$
-  add_data(x = m2, startCol = 4)$
+  add_data(x = m1)$
+  add_data(x = m2, dims = wb_dims(start_col = 4))$
```

Comment:
Oh wow, impressive. I'd never assumed this, but wow this way we could really remove start_col/start_row arguments from the function. Still, I want something like #469 sometime in the future. Adding checks if `x` fits into `dims`. If a user wants to write `matrix(1, 5, 5)` into `dims = "A1:B2"` we should return a warning.

Reply:
Possibly.. 

I am not experienced enough to do that I believe, but this would be a good idea!

Reply:
For a basic check it would be sufficient to check if `dim(dims_to_dataframe(dims)) == dims(x)` where x includes row_names and col_names, but ... it always felt a bit to much and ... there is no other explanation :)